service/health/searcher/kvm.go

```diff
@@ -2,12 +2,30 @@ package searcher
 
 import (
 	"context"
+
+	"github.com/giantswarm/microerror"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // searchKVMCR searches for the cluster config in AWSClusterConfigs resources.
 func (s *Service) searchKVMCR(ctx context.Context, request Request) (*Response, error) {
+	kvmCR, err := s.G8sClient.ProviderV1alpha1().KVMConfigs("default").Get(request.ClusterID, v1.GetOptions{})
```

Comment:
Let's do this in a separate PR and get the basic unit tests merged first.

Reply:
Okay, will branch and revert.

Reply:
Moved to https://github.com/giantswarm/health-service/tree/kvm-health